modules/300-prometheus/crds/prometheusremotewrite.yaml

```diff
@@ -56,6 +56,14 @@ spec:
                 customAuthToken:
                   type: string
                   description: Custom token sent as a value of the X-Auth-Token header.
+                headers:
+                  description: |
+                    Custom HTTP headers to be sent along with each remote write request. Alloewd headers: X-scope-OrgID.
+                  type: object
+                  patternProperties:
+                    "^X-Scope-OrgID$": 
```

Comment:
Why are we using `patternProperties` here? Since the regular expression matches the whole fixed string, why not just use `properties`?

Reply:
The main idea is that in the future, we can easily extend the list of allowed headers by simply updating the regular expression.